checkov/common/bridgecrew/integration_features/features/policy_metadata_integration.py

```diff
@@ -0,0 +1,105 @@
+import logging
+import re
+
+from checkov.common.bridgecrew.integration_features.base_integration_feature import BaseIntegrationFeature
+from checkov.common.bridgecrew.platform_integration import bc_integration
+from checkov.common.bridgecrew.severities import Severities
+from checkov.common.checks.base_check_registry import BaseCheckRegistry
+
+# service-provider::service-name::data-type-name
+
+CFN_RESOURCE_TYPE_IDENTIFIER = re.compile(r"^[a-zA-Z0-9]+::[a-zA-Z0-9]+::[a-zA-Z0-9]+$")
+
+
+class PolicyMetadataIntegration(BaseIntegrationFeature):
+    def __init__(self, bc_integration):
+        super().__init__(bc_integration, order=0)
+        self.check_metadata = {}
+        self.bc_to_ckv_id_mapping = {}
+
+    def is_valid(self) -> bool:
+        return (
+            not self.bc_integration.skip_download
+            and not self.integration_feature_failures
+        )
+
+    def pre_scan(self):
+        try:
+            if self.bc_integration.customer_run_config_response:
+                self._handle_customer_run_config(self.bc_integration.customer_run_config_response)
+            elif self.bc_integration.public_metadata_response:
+                self._handle_public_metadata(self.bc_integration.public_metadata_response)
+            else:
+                logging.warning('In the pre-scan for policy metadata, but nothing was fetched from the platform')
```

Comment:
No, because if the platform integration isn't configured, this won't run. But I think I can bump the level down to debug, as we always just get debug logs whenever there is unexpected behavior anyways.